docs/automated-workflows/dagman-reference.rst

```diff
@@ -348,6 +348,23 @@ description file.
 
     $ condor_submit_dag diamond.dag
 
+If a DAG was already executed before and had previously failed then upon
+re-submission of the DAG, DAGMan will locate the most recent Rescue DAG file
```

Comment:
This seems a bit wordy.  Can we try something like

When `condor_submit_dag` starts, it checks to see if a previous attempt to run this dag failed.  If so ...